lib/engine.py

```diff
@@ -102,11 +102,16 @@ def source_gen():
 
         total = self.table.record_id + real_line_length
         pos = 0
+        count_iter = 1
+        insert_limit = 200
+        current = 0
+        types = self.table.get_column_datatypes()
+        multiple_values = []
         for line in real_lines:
             if not self.table.fixed_width:
                 # This replaces end of line characters that exist in a single line
                 # eg. "one \nline has multiple end of lines\n"
-                line = line.replace('\n', '').strip()
+                line = line.strip()
```

Comment:
Can you walk me through why we're removing this here? Since we're pulling the `replace` the comment should also be removed since it is just explaining the `replace`.